Source/Scene/Cesium3DTileOptimizations.js

```diff
@@ -0,0 +1,125 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/defaultValue',
+        '../Core/freezeObject',
+        './TileOrientedBoundingBox',
+        './TileBoundingRegion'
+    ], function(
+        Cartesian3,
+        defaultValue,
+        freezeObject,
+        TileOrientedBoundingBox,
+        TileBoundingRegion) {
+    'use strict';
+
+    /**
+     * @alias Cesium3DTileOptimizationFlag
+     *
+     * Flag denoting support for a 3D Tiles optimization
+     */
+    var Cesium3DTileOptimizationFlag = {
```

Comment:
This should also be `@private` in its own file.

Also, we generally don't suffix enums with `Flag` or `Flags`.  Perhaps there are better names for these.